app/common/application/resources/app.json

```diff
@@ -3,7 +3,7 @@
       {"name":"APP_REGISTRATION", "url":"/registration", "logo":"images/hospital.png", "active": true},
       {"name":"APP_CLINIC", "url":"/clinic", "logo":"images/clinic.png", "active": true},
       {"name":"APP_VITALS", "url":"/vitals", "logo":"images/vitals.png", "active": true},
-      {"name":"APP_PHARMACY", "url":"/pharmacy", "logo":"images/medical.png", "active": false},
+      {"name":"APP_PHARMACY", "url":"/pharmacy", "logo":"images/medical.png", "active": true},
```

Comment:
I think since we have NOT implemented this, we have to set last parameter to false.